### PR TITLE
Adding LD_LIBRARY_PATH to GAMS installer bash script

### PR DESCRIPTION
**Pull request by mrmundt**
Pyomo's Travis builds started to run very slowly. This fixes that issue.

### CODE REVIEW

File: linux_install_scripts/gams.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ ARG TARGET="linux_x64_64_sfx.exe"
 ARG GAMS_DIR="gams29.1_linux_x64_64_sfx"
 ENV GAMS_VERSION="29.1.0"
 ENV PATH="${PREFIX}/GAMS_${GAMS_VERSION}/${GAMS_DIR}:${PATH}"
+ENV LD_LIBRARY_PATH="${PREFIX}/GAMS_${GAMS_VERSION}/${GAMS_DIR}"
```
Comment (jsiirola) [inline linux_install_scripts/gams.sh]: For safety / future proofing, this needs to augment the LD_LIBRARY_PATH env var and not just overwrite it (like the PATH variable above).

**Review by jsiirola — CHANGES_REQUESTED** [review]
One minor change (do not overwrite the env var)

**Review by mrmundt — COMMENTED** [review]
Changed.

**Review by jsiirola — APPROVED** [review]

**Comment by jsiirola** [comment]
This is a minor change, and at least one test has passed.  I am going to merge (this will significantly ease the Travis backlog!).

**Pull request merged**